server/src/main/java/org/elasticsearch/search/aggregations/metrics/MinAggregator.java

```diff
@@ -181,7 +181,7 @@ public void doClose() {
         if (parent != null) {
             return null;
         }
-        if (config.fieldContext() != null && config.script() == null) {
+        if (config.fieldContext() != null && config.script() == null && config.missing() == null) {
```

Comment:
this change looks unrelated?

Reply:
Bug actually, stumbled into it while working on the PR and made a tweak.  Since we wrap the DV with an anonymous class that injects missing values, and the BKD optimization aborts before we use those DVs, we can miss a potential min/max value if it happened to be the missing value.

@imotov just fixed this in https://github.com/elastic/elasticsearch/pull/48970 due to a bug report, and better to have it fixed separately anyway so this part of the change can go away.